ResearchKitTests/ORKTextChoiceCellGroupTests.m

```diff
@@ -55,6 +55,19 @@ - (NSArray *)textChoices {
     return choices;
 }
 
+- (NSArray *)textChoicesWithExclusive {
+    static NSArray *choicesWithExclusive = nil;
+    
+    if (choicesWithExclusive == nil) {
+        choicesWithExclusive = @[[ORKTextChoice choiceWithText:@"choice 01" value:@"c1"],
+                                [ORKTextChoice choiceWithText:@"choice 02" value:@"c2" exclusive:YES],
+                                [ORKTextChoice choiceWithText:@"choice 03" value:@"c3"],
+                                [ORKTextChoice choiceWithText:@"choice 04" value:@"c4"]];
```

Comment:
I've added a test for all choices being exclusive, which I reused the existing single choice test for, since it should act the same.  I also added some code to handle testing a set of choices with two exclusives.